src/components/select2/AuiSelect2Single.vue

```diff
@@ -42,7 +43,9 @@
         data: () => ({results: this.options}),
         placeholder: this.placeholder,
         width: this.width,
-        dropdownAutoWidth: this.dropdownAutoWidth
+        dropdownAutoWidth: this.dropdownAutoWidth,
+        formatResult: (state) => state.text,
+        formatSelection: (state) => state.text
```

Comment:
Other places refer to these objects as `options` so please rename to `(option) -> ...` to be consistent.

Reply:
Wait. I don't understand one thing. Currently, (if I understand well from select2 docs, haven't tested yet), this is the default behaviour, so does those 2 lines do anything?
And 2nd, why not just use the passed renderer here similar to this
    
    formatSelection: this.formatResult || (option) => option.text

won't this work?

Reply:
Yes, I will try it